debian/control

```diff
@@ -6,22 +5,30 @@ Maintainer: Go Compiler Team <pkg-golang-devel@lists.alioth.debian.org>
 Uploaders: Michael Stapelberg <stapelberg@debian.org>,
            Paul Tagliamonte <paultag@debian.org>,
            Tianon Gravi <admwiggin@gmail.com>
-Vcs-Browser: http://anonscm.debian.org/gitweb/?p=pkg-golang/golang.git
+Vcs-Browser: https://anonscm.debian.org/cgit/pkg-golang/golang.git
 Vcs-Git: git://anonscm.debian.org/pkg-golang/golang.git
-Build-Depends: debhelper (>= 7.4.10), bison, ed, mawk | awk, perl, netbase
+Build-Depends: bison,
+               debhelper (>= 7.4.10),
+               ed,
+               golang (>= 1:1.4~) | gccgo-5,
```

Comment:
Why would this FTBFS?  If `golang` 1.4 or greater is available (current version in sid), we compile with that.  If it's not, we grab `gccgo-5`, which provides a compatible compiler, and is only used for bootstrapping `golang` 1.5+.